src/clr-angular/data/datagrid/datagrid.html

```diff
@@ -15,7 +15,8 @@
                         <div role="columnheader" class="datagrid-column datagrid-select datagrid-fixed-column"
                              *ngIf="selection.selectionType === SELECTION_TYPE.Multi">
                         <span class="datagrid-column-title">
-                            <clr-checkbox [(ngModel)]="allSelected"></clr-checkbox>
+                            <input clrCheckbox type="checkbox" [(ngModel)]="allSelected"
```

Comment:
Thanks for updating to the new forms!

Reply:
We haven't done the new checkboxes yet...